docs/slack-apps/tutorials/channel-check.md

```diff
@@ -0,0 +1,70 @@
+
+## Building in Dark
+
+1. Create a new daily CRON named checkForNewChannels by clicking the + in the sidebar next to Cron, and call the [getNewConversations Slack function](../slack-packages#getnewconversations) (it's a little confusing, but channels are referred to as conversations in the Slack API). The getNewConversations function is built into Dark's [package manager](./../../packages), and it takes your token as well as an amount of time to check in Epoch seconds. Here, we're asking it to check in the past 24 hours, which is equivalent to 86400 seconds.
+
+![assets/slack-apps/tutorials/channelcheck/checkfornewchannels.png](assets/slack-apps/tutorials/channelcheck/checkfornewchannels.png)
+
+2. Click the play button on the [getNewConversations function](../slack-packages#getnewconversations) and then place your carat in newChannels. You will see a list of information about any new channels that have been created.
+
+![assets/slack-apps/tutorials/channelcheck/checkfornewchannelslive.png](assets/slack-apps/tutorials/channelcheck/checkfornewchannelslive.png)
+
+3. Create a function called formatChannelList with a channelList parameter. This is where we're going to build the format of the messages Slack will print about your new channels, but for now, just return channelList. You'll notice that an error is shown - this is because channelList has not yet been set.
+
+![assets/slack-apps/tutorials/channelcheck/firstformatchannel.png](assets/slack-apps/tutorials/channelcheck/firstformatchannel.png)
+
+4. Return to your checkForNewChannels CRON, and add a line to format the newChannels list you previously generated. Press the play button.
+
+![assets/slack-apps/tutorials/channelcheck/checkfornewchannelslist.png](assets/slack-apps/tutorials/channelcheck/checkfornewchannelslist.png)
+
+5. Return to your formatChannelList function and place your carat over the top dot (trace) on the left. You will see your channelList function - now you can work with that data in order to format your list.
+
+![assets/slack-apps/tutorials/channelcheck/formatchannellist.png](assets/slack-apps/tutorials/channelcheck/formatchannellist.png)
+
+6. For our purposes, we're going to do a very simple amount of formatting - we're going to add the markdown Slack needs to make channel names links, and add information about the channel's purpose. However, lots of information about Slack channels is available. Type `val.` and scroll through the autocomplete to see all of your options. 
+
+![assets/slack-apps/tutorials/channelcheck/formatchanellistlong.png](assets/slack-apps/tutorials/channelcheck/formatchannellistlong.png)
+[Function on Sample Canvas](https://darklang.com/a/sample-slackchannelcheck#fn=1331376176)
```

Comment:
The caption here is confusing. Same with all following captions, I'd recommend removing them.